web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -15,7 +15,6 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     PENDING_EVENT_ATTR: 'PENDING_EVENT',
 
     constructor: function(cfg) {
-
```

Comment:
From now on BTW, my 'developer discretion' will be to have 10 empty lines at the end of every function implementation.